### PR TITLE
ci: upgrade deprecated actions

### PR DESCRIPTION
**Pull request by whynowy**
Explain what this PR does.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -137,32 +137,31 @@ jobs:
     timeout-minutes: 20
     strategy:
       fail-fast: false
-      max-parallel: 13
```
Comment (whynowy) [inline .github/workflows/ci.yaml:140]: Maximumly use the resources.

**Review by vigith — APPROVED** [review]

**Comment by vigith** [comment]
We need to have a different strategy for redis since most of them are Skip. The current setup is very slow.

**Comment by whynowy** [comment]
> We need to have a different strategy for redis since most of them are Skip. The current setup is very slow.

true.

**Comment by vigith** [comment]
> > We need to have a different strategy for redis since most of them are Skip. The current setup is very slow.
> 
> true.

Disable Redis for the time being?

**Comment by whynowy** [comment]
> > > We need to have a different strategy for redis since most of them are Skip. The current setup is very slow.
> > 
> > 
> > true.
> 
> Disable Redis for the time being?



> > > We need to have a different strategy for redis since most of them are Skip. The current setup is very slow.
> > 
> > 
> > true.
> 
> Disable Redis for the time being?

Partial enable.

**Pull request merged**